plugins/exporter.koplugin/target/markdown.lua

```diff
@@ -0,0 +1,160 @@
+local _ = require("gettext")
+local UIManager = require("ui/uimanager")
+
+local formatters = {
+    none = "%s",
+    bold = "**%s**",
+    italic = "*%s*",
+    bold_italic = "**_%s_**",
+    underline_markdownit = "++%s++",
+    underline_u_tag = "<u>%s</u>",
+    strikethrough = "~~%s~~",
+}
+
+-- markdown exporter
+local MarkdownExporter = require("base"):new {
+    name = "markdown",
+    extension = "md",
+}
+
+function MarkdownExporter:loadSettings()
+    local plugin_settings = G_reader_settings:readSetting("exporter") or {}
+    self.settings = plugin_settings[self.name] or {}
+end
+
+local formatter_buttons = {
+    {_("None"), "none"},
+    {_("Bold"), "bold"},
+    {_("Bold Italic"), "bold_italic"},
+    {_("Italic"), "italic"},
+    {_("Strikethrough"), "strikethrough"},
+    {_("Underline (Markdownit style, with ++)"), "underline_markdownit"},
+    {_("Underline (with <u></u> tags)"), "underline_u_tag"},
+}
+
+function MarkdownExporter:editFormatStyle(drawer_style)
+    local radio_buttons = {}
+    for _idx, v in ipairs(formatter_buttons) do
+        table.insert(radio_buttons, {
+            {
+                text = v[1],
+                checked = self.settings.formatting_options[drawer_style] == v[2],
+                provider = v[2],
+            },
+        })
+    end
+    UIManager:show(require("ui/widget/radiobuttonwidget"):new{
+        title_text = _("Formatting style for ")  .. drawer_style,
```

Comment:
Nit: double space.
Maybe it is better to use labels' text (already translated strings) instead of the internal names for `drawer_style`.

Reply:
Tried to fix this. Is it okay now?